addons/respawn/functions/fnc_adjustTimeLocal.sqf

```diff
@@ -0,0 +1,27 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3
+ * Function adjusts new remaining respawn time with respect to current elapsed time.
+ *
+ * Arguments:
+ * 0: New respawn time delay <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [30] call afm_respawn_fnc_adjustTimeLocal
+ *
+ * Public: No
+ */
+
+params ["_newTime"];
+
+if (playerRespawnTime isEqualTo -1) exitWith {};
```

Comment:
If we exit there `GVAR(oldTime)` will be not adjusted for alive players. If I understand the flow correctly doing `GVAR(oldTime) = GVAR(time)` in `onPlayerKilled` would fix the issue.

Reply:
I think it would be better to just set this `GVAR(oldTime)` in exit also, good catch. No point in setting this every time someone gets killed as this won't change often.